Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -118,6 +123,8 @@ public void Commit()
                     MeshObject.SetTriangles(tris, 0);
                     MeshObject.RecalculateNormals();
                     MeshObject.RecalculateBounds();
+                    SpatialCollider.sharedMesh = null;
+                    SpatialCollider.sharedMesh = MeshObject;
```

Comment:
You're correct. I had to look it up. Strange that you'd need to set it null before setting it, but it's the only way to reset the shared mesh collider apparently.